### DISCUSSION TITLE
Search show only results when file name starts with the search phrase

### DISCUSSION THREAD

**Discussion by joanlel** [microsoft-search/pnp-modern-search#3737] [General]
**Version used**
4.11

**Describe the bug**
Search results show only files, which file name starts with the search phrase, but not when the name contains it.

**To Reproduce**
Detailed steps to reproduce the behavior:
I use search results + search box. 
Query template:
(Filename:{searchTerms}*) -ContentTypeId:0x0120* IsDocument:1 path:https://xxxxxxx.sharepoint.com/sites/xxxxxx/xxxx
File name that I test: "MovingTargets"
When I enter "Move"/ "Mo"/"Movin"/"Moving" etc the file is shown.
When I enter "Targets" - or something from the middle of the word, the file will be not found.

**Expected behavior**
Search results shows every file, where the file name contains search phrase.

**Desktop (please complete the following information):**
 - Browser chrome, mozilla

**Comment by kasperbolarsen** [comment]
Hi,
That is the way the query language works. "Ends with" is not possible, see https://learn.microsoft.com/en-us/sharepoint/dev/general-development/keyword-query-language-kql-syntax-reference
